src/test/e2e/cypress/integration/logged-out-user-can-submit-feedback.js

```diff
@@ -0,0 +1,9 @@
+describe("As a logged out user", () => {
+  it("I can submit feedback about the service via the feedback form", () => {
+    cy.visit("/");
+    cy.get("a").contains("feedback").click();
+    cy.get('[type="radio"]').first().check();
+    cy.get("textarea").type("Nothing, it's amazing.");
+    cy.get("form").submit();
```

Comment:
After months of custom abstractions selectors and assertions (`givenIHaveSelectedAMaritimeUse()`), what do we think about using plain ol' Cypress in our (much lighter) end-to-end tests?  I've got to admit, it was nice just to write some Cypress methods here.